index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Africa Day concert 2023</title>
+    <link rel="stylesheet" href="./assets/css/home.css">
+    <script src="https://kit.fontawesome.com/1f634f9b31.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="set-background">
+    <header class="header">
+        <nav class="social-media" aria-label="social media icons">
+            <ul class="social-media-ul">
+                <li><a href=""><img src="./assets/images/social-media-images/Facebook.svg" alt=""></a></li>
+                <li><a href=""><img src="./assets/images/social-media-images/twitter.svg" alt=""></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="https://github.com/yusufyusice">My github page</a></li>
+                <li><a href="#">Logout</a></li>
```

Comment:
- [ ] kindly consider making sure that this nav is on the same straight line as its below navbar to make your design look exactly as the given one 
- your design to be fixed 👇 
<img width="1330" alt="Screen Shot 2023-06-30 at 23 24 33" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/60146030/3c5f5d87-fd58-4837-b1a6-dcca17785d59">